packages/desktop-client/src/components/settings/Experimental.js

```diff
@@ -53,6 +55,19 @@ export default function ExperimentalFeatures({ prefs, savePrefs }) {
               />{' '}
               <View>Enable account syncing</View>
             </label>
+            <label style={{ display: 'flex' }}>
+              <Checkbox
+                id="report-budget-flag"
```

Comment:
Oop— can you give the `id`s unique values?

Reply:
I will.